Testscripts/Linux/TIME-CLOCKSOURCE.sh

```diff
@@ -26,14 +26,22 @@ UtilsInit
 CheckSource()
 {
 	current_clocksource="/sys/devices/system/clocksource/clocksource0/current_clocksource"
+	# Microsoft LIS installed version has lis_hyperv_clocksource_tsc_page
+	# Without Microsoft LIS, hyperv_clocksource_tsc_page
+	# CentOS 6.8 or older versions, hyperv_clocksource 
 	clocksource="hyperv_clocksource_tsc_page"
+	mj=$(echo $DISTRO_VERSION | cut -d '.' -f 1)
```

Comment:
Do this under the check of DISTRO == rh | centos

Reply:
Fixed and verified in the same set of rhel/centos along with SLES 15-SP2 and Ubuntu 18.04